### PR TITLE
Fixed typo in eda notebook.

### PR DESCRIPTION
**Pull request by kassyray**
# Proposed Changes 

I changed `Extract the cleaned ECG signal` to `Extract the cleaned EDA signal` in the `Analyze EDA Notebook` because this notebook deals with EDA signals, not ECG. This was a small typo, but can be misleading when trying to follow the tutorial.

### CODE REVIEW

**Comment by DominiqueMakowski** [comment]
Alright, @kassyray let's consider this as your first/test PR 😁  

So as the project is growing we're trying to adopt a more fleshed-out and structured development workflow. Essentially, that means we have two branches, a dev and a master one, and we try not to commit directly to the master branch, but to direct the PR to the dev one.

So I know that it is a bit irritating at first to get the Github workflow right, and to me, it was always super hard to get used to it (and it still is). Nevertheless, we believe acquiring these skills and using these good practices are for a good cause in the end.

Long story short, this PR is currently directed at the master branch. Could you 1) close this PR (i.e., cancel it) and 2) make a new PR, that will be directed at the dev branch? That involves placing yourself on the dev and then making a PR from there.

A help doc here (let us know if it helps and feel free to improve it based on your experience!): https://neurokit2.readthedocs.io/en/latest/contributing/contributing.html#development-workflow

Let us know if you have any question!

**Comment by kassyray** [comment]
Thanks @DominiqueMakowski 😊

I **think** (and hope) I followed the correct workflow this time - a new PR has now been directed to the dev branch. 

Thank you for being so helpful 😄
